src/stactools/sentinel2/stac.py

```diff
@@ -93,25 +107,27 @@ def create_item(
 
     # Metadata
 
-    item.add_asset(*safe_manifest.create_asset())
+    if safe_manifest is not None:
```

Comment:
this code was giving warnings about safe_manifest possibly not being defined, so I put these in this wrapper for when we add the ability to not need the safe manifest.